vms/proposervm/vm.go

```diff
@@ -845,7 +671,7 @@ func (vm *VM) acceptPostForkBlock(blk PostForkBlock) error {
 	if err := vm.State.PutBlock(blk.getStatelessBlk(), choices.Accepted); err != nil {
 		return err
 	}
-	if err := vm.updateHeightIndex(height, blkID); err != nil {
+	if err := vm.storeHeightEntry(height, blkID); err != nil {
```

Comment:
this is the only part of `updateHeightIndex` that remains after this PR, so substituting calling `storeHeightEntry` directly here.

Alternatively, can rename `storeHeightEntry` to `updateHeightIndex` so this is unchanged.